src/benchmarks/benchmark.h

```diff
@@ -106,14 +108,11 @@ struct benchmark_results
 /*
  * Command Line Option integer value base.
  */
-enum clo_int_base
```

Comment:
Turn this to a class with a static method (for string conversion) and a single enum value. Or, if you feel comfortable with it, use the enum as a template argument for a function.